examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -450,6 +450,9 @@ CHIP_ERROR CommandHandler(int argc, char ** argv)
 
         targetCastingPlayer->VerifyOrEstablishConnection(connectionCallbacks, matter::casting::core::kCommissioningWindowTimeoutSec,
                                                          idOptions);
+        ChipLogProgress(AppServer, "CommandHandler() request, VerifyOrEstablishConnection() called, calling StopDiscovery()");
+        // Stop discovery since we have found are connecting to the desired CastingPlayer.
```

Comment:
Is there a typo in this comment?

Reply:
Yes
// Stop discovery since we have discovered, and are now connecting to the desired CastingPlayer.